### PR TITLE
CIF-831 - Make AEM GraphQL "authoring" connector opensource

### PR DESCRIPTION
**Pull request by cjelger**

### CODE REVIEW

File: all/pom.xml [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+                        </subPackage>
+                        <subPackage>
+                            <groupId>${project.groupId}</groupId>
+                            <artifactId>cif-virtual-catalog-content</artifactId>
+                            <filter>true</filter>
+                        </subPackage>
+                    </subPackages>
+                </configuration>
+            </plugin>
+            <plugin>
```
Comment (mhaack) [inline all/pom.xml]: Is this plugin still needed?

**Review by dplaton — APPROVED** [review]

File: content/cif-connector/pom.xml [hunk 2]
```diff
@@ -0,0 +1,304 @@
...
+                                    <filtering>false</filtering>
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+            <plugin>
+                <groupId>com.day.jcr.vault</groupId>
```
Comment (mhaack) [inline content/cif-connector/pom.xml:113]: This should use the `filevault-package-maven-plugin` to build the package as well I think.

File: content/cif-virtual-catalog/pom.xml [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+                                </resource>
+                            </resources>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            
+            <plugin>
+                <groupId>com.day.jcr.vault</groupId>
+                <artifactId>content-package-maven-plugin</artifactId>
```
Comment (mhaack) [inline content/cif-virtual-catalog/pom.xml:115]: This should be using the `filevault-package-maven-plugin` as well.

File: all/pom.xml (hunk 1 above)
Comment (cjelger) [reply all/pom.xml]: Yes, this is required to be able to do `content-package:install`

Comment (mhaack) [reply all/pom.xml]: Oh right.

**Comment by cjelger** [comment]
@mhaack actually I removed the `filevault` plugin and replaced it with the content package plugin ... I had copied that from Mark, I don't know what the difference is between the two, they seem to do exactly the same.

**Review by mhaack — APPROVED** [review]

**Pull request merged**